workflow/artifacts/gcs/gcs.go

```diff
@@ -0,0 +1,131 @@
+package gcs
+
+import (
+	"cloud.google.com/go/storage"
+	"context"
+	"errors"
+	"fmt"
+	argoErrors "github.com/argoproj/argo/errors"
+	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/api/option"
+	"io"
+	"os"
+)
+
+type GCSArtifactDriver struct {
+	Context       context.Context
+	CredsJSONData []byte
+}
+
+func (gcsDriver *GCSArtifactDriver) newGcsClient() (client *storage.Client, err error) {
+	gcsDriver.Context = context.Background()
+
+	fmt.Println(string(gcsDriver.CredsJSONData))
```

Comment:
this is a debugging line that i accidentally left in and it absolutely needs to be removed